ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/party_management_service.proto

```diff
@@ -129,10 +177,15 @@ message PartyDetails {
 
   // Human readable name associated with the party. Caution, it might not be
   // unique.
+  // Use of this field is discouraged. Consider using modifiable participant local party metadata instead.
   // Optional
   string display_name = 2;
 
   // true if party is hosted by the backing participant.
   // Required
```

Comment:
It's not required when updating party details is it?

Reply:
> I'm not thinking that it's weird that a boolean field is required. Isn't it equivalent of saying that it must always be set to true?

I understand 'Required' to mean that the server must always set the field to its actual value; and thus, the client can rely on its value. So if the value is false, then the field is false, which happens to be default, but that doesn't matter. The important part is that for responses where the party is hosted on the backing participant `is_local` is set to `true`
 
> Hmm, is it that `Required`/`Optional` notation is only meant for requests, and is not applicable at all to responses?

Reply:
I reviewed usage of 'Required'/'Optional' in UM and PM and added an explanation what I understand it means [here](https://github.com/digital-asset/daml/pull/14550/files#r943431284) -